config/default/llmisvcconfig_validatingwebhook_cainjection_patch.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: llminferenceserviceconfig.serving.kserve.io
+  annotations:
+    cert-manager.io/inject-ca-from: $(kserveNamespace)/serving-cert
+webhooks:
+  - name: llminferenceserviceconfig.kserve-webhook-server.validator
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)